sdk/nodejs/src/client.ts

```diff
@@ -296,6 +303,35 @@ export class DevCycleClient {
         this.eventQueue.queueEvent(populatedUser, event)
     }
 
+    private trackSDKConfigEvent(
+        url: string,
+        responseTimeMS: number,
+        res?: Response,
+        err?: ResponseError,
+        reqEtag?: string,
+        reqLastModified?: string,
+    ): void {
+        this.eventQueue.queueEvent(
+            DVCPopulatedUserFromDevCycleUser({ user_id: this.clientUUID }),
+            {
+                type: 'sdkConfig',
+                target: url,
+                value: responseTimeMS,
+                metaData: {
+                    clientUUID: this.clientUUID,
+                    reqEtag,
+                    reqLastModified,
+                    resEtag: res?.headers.get('etag') || undefined,
+                    resLastModified:
+                        res?.headers.get('last-modified') || undefined,
+                    resRayId: res?.headers.get('cf-ray') || undefined,
+                    resStatus: err ? err.status : res?.status || undefined,
+                    errMsg: err ? err.message : undefined,
```

Comment:
is this gonna be the right message? like I think this is just getting the exception message from the error object and not actually the response error data?

Reply:
yea all the response errors are wrapped in this: 
<img width="655" alt="image" src="https://github.com/DevCycleHQ/js-sdks/assets/1219069/a6c79a4a-88d5-467c-8f18-129fd8940943">

so `err.message` should be correct.

Reply:
yeah but the message there is still the exception message and wouldnt contain any data about the response itself

Reply:
looks like it would be the `message` field returned in the json body of the response if that's parseable: 
<img width="744" alt="image" src="https://github.com/DevCycleHQ/js-sdks/assets/1219069/24cbc547-5335-451b-9249-df1220dccc1c">

Reply:
ok. keep in mind cloudflare doesnt have JSON responses on the CDN though, its gonna be html. So its likely this will just always say "Something went wrong"

Reply:
yea I think that's ok for now, I don't think we are relying on this for error tracking that way. The status is all we really need.